src/mongo/Config.class.php

```diff
@@ -247,11 +247,26 @@ protected function loadConfig(array $config)
                     // Ensure indexes are legal
                     if (array_key_exists("indexes", $podConfig)) {
                         $this->indexes[$storeName][$podName] = [];
+
+                        error_log('Config indexes: '.json_encode($podConfig["indexes"]));
                         foreach ($podConfig["indexes"] as $indexName => $indexFields) {
+
+                            $this->indexes[$storeName][$podName][$indexName] = $indexFields;
+                            error_log('Index for store: '.$storeName.' - '.json_encode($indexFields));
+
+                            $indexKeys = array_keys($indexFields);
+                            if (is_numeric($indexKeys[0])) {
```

Comment:
[question] could this logic be pulled out to a well named function to help explain it's intention?